### ISSUE TITLE
[Feature Request] Auto stash when start feature/release/hotfix

### ISSUE THREAD

**Issue by Rwing** [ForkIssues/TrackerWin#693]
as title

**Comment by jerone** [comment]
Agree. Every time I start a new feature, with already changes, I get the following error:

![image](https://user-images.githubusercontent.com/55841/75871531-fddda000-5e0c-11ea-97e4-aa696ac9cb80.png)

**Comment by jerone** [comment]
Duplicate issue with error: #633
